Makefile

```diff
@@ -110,6 +111,10 @@ server-static: build-web
 	rm -f $(BIN_OUTPUT_PATH)/viam-server
 	VIAM_STATIC_BUILD=1 GOFLAGS=$(GOFLAGS) go build $(LDFLAGS) -o $(BIN_OUTPUT_PATH)/viam-server web/cmd/server/main.go
 
+full-static:
```

Comment:
This needs to depend on `build-web` so the front end code gets included.

Reply:
done in ea5cebd63, sample run [here](https://github.com/viamrobotics/rdk/actions/runs/8883537628/job/24390599038)